packages/server/src/client/open-feature-client.ts

```diff
@@ -278,6 +279,16 @@ export class OpenFeatureClient implements Client, ManageContext<OpenFeatureClien
         flagKey,
       };
 
+      if (evaluationDetails.errorCode || evaluationDetails.reason === StandardResolutionReasons.ERROR) {
+        const errorCode = evaluationDetails.errorCode || ErrorCode.GENERAL;
+        await this.errorHooks(allHooksReversed, hookContext, instantiateErrorByErrorCode(errorCode), options);
+        return {
+          ...evaluationDetails,
```

Comment:
Should we add our own `errorMessage` if one wasn't supplied by the provider?

I suspect in most cases they would populate one, and if we added one it might not be very informative... we could add something like `"provider returned error code"`... Leaning towards no, but wanted to mention it.

Reply:
I don't think that provides any additional information that would be useful. If the provider sets an error message, it will be retained, but setting a default message would basically rephrase the same information that's already available.